README.md

```diff
@@ -6,6 +6,28 @@ Administration comprises starting/stopping the host, configuring modules and the
 
 In-game management (e.g. removing users from a running game) should be performed through users with admin rights.
 
+### Setting up from source
+
+This facade can be set up using Gradle as explained in the [Terasology wiki](https://github.com/MovingBlocks/Terasology/wiki/Codebase-Structure#facades).
+Here is a step-by-step summary:
+* `cd` to the root of your Terasology workspace (the directory where you cloned the main repository [MovingBlocks/Terasology](https://github.com/MovingBlocks/Terasology)).
+* Execute `./gradlew fetchFacadeServer`; it will automatically clone this repository in the `facades/Server` directory.
+* Keep in mind that while the default branch on GitHub is `develop`, Gradle pulls the `master` branch. So to use the latest available version or work with it, you may consider switching to the `develop` branch:
+    * `cd facades/Server`
+    * `git checkout develop`
+    * `cd ../..`
+* Run `./gradlew jar`
+* To start a server using FacadeServer, execute `./gradlew facades:Server:run`.
+
+### API documentation
+
+The web server exposes access to the various resources both with an HTTP REST API and over WebSocket.
+More information is available in the docs.md file at the root of this repository.
+Also, here is a [direct link](http://petstore.swagger.io/?url=https://raw.githubusercontent.com/gianluca-nitti/FacadeServer/rest-methods/swagger.json#/) to view online the Swagger/OpenAPI specification of the HTTP API.
```

Comment:
Sorry for the very late response, I didn't manage to answer your comments before this pull request was merged. Anyway, if necessary I may add changes related to this to the new PR #10.

I used this link because it directs to an interface meant for *viewing* the API doc, which in my understanding is the goal of this link; the editor page shows a split-screen interface which is the same of the petstore on one side and a text editor for the specification file on the other one. Obviously this wouldn't allow people to actually edit the specification file in the repository because it's just fetched through an HTTP GET from `raw.githubusercontent.com`, but anyway I didn't find it ideal for someone which simply needs to view the documentation. This is just my opinion though, if you prefer the editor version I'll switch the link to that one.

Comparison: [Petstore link (currently in the readme file)](http://petstore.swagger.io/?url=https://raw.githubusercontent.com/MovingBlocks/FacadeServer/develop/src/main/resources/web/swagger.json#/) - [Editor link](http://editor.swagger.io/?url=https://raw.githubusercontent.com/MovingBlocks/FacadeServer/develop/src/main/resources/web/swagger.json#/)